### PR TITLE
ENG-2677 Fix content config templates fetching

### PR DESCRIPTION
**Pull request by raxkaynan**
Change used action for fetching content templates to avoid getting existing filter state

### CODE REVIEW

**Review by jeffgo10 — APPROVED** [review]
looks good

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-cms/pull/427#pullrequestreview-780375428" title="Approved">jeffgo10</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-cms/blob/release/0.0.278/OWNERS)~~ [jeffgo10]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
